src/yarpdatadumper/main.cpp

```diff
@@ -130,7 +138,23 @@ class DumpImage : public DumpObj
         return (fName.str()+" ["+Vocab::decode(code)+"]");
     }
 
-    void *getPtr() override { return p->getRawImage(); }
+    const cv::Mat &getImage()
```

Comment:
If I understood correctly `yarpdatadumper` should be usable also without `OpenCV`, in this sense this function should be included in `#ifdef ADD_VIDEO #endif` guards.

Reply:
You're absolutely right! Thanks for pointing it out 👍